serializers.py

```diff
@@ -1416,6 +1416,8 @@ def before_validation(self, data: dict):
         if not data.get("date_due"):
             data["date_due"] = data.get("date_completed") or data.get("date_assigned")
 
+        data["date_due"] = datetime.strptime(data["date_due"], '%Y-%m-%dT%H:%M:%S%z').strftime('%Y-%m-%d')
```

Comment:
Does the due date in OJS contain time component as well or is the time component an artifact of serialization?

Reply:
the due date in the ojs api has the time component at this time. I can remove it but it was getting re-added in the serialization process. Also, the line above this is assigning "date_completed" or "date_assigned" to due date if due_date doesn't exist.  those are both legitimately datetimes.